### PR TITLE
Remove 1Password backend support and legacy code... feels good!

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Just, yeah, `4,962 deletions`.

### CODE REVIEW

File: config/crd/bases/store.externalsecret-operator.container-solutions.com_secretstores.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
+    controller-gen.kubebuilder.io/version: v0.2.5
```
Comment (knelasevero) [inline config/crd/bases/store.externalsecret-operator.container-solutions.com_secretstores.yaml:7]: Why are we changing this version here?

Comment (jonatasbaldin) [reply config/crd/bases/store.externalsecret-operator.container-solutions.com_secretstores.yaml:7]: I think I did a `make manifests` and it updated the CRDs. I'll revert this, one sec.

**Pull request merged**